cmd/poktrolld/cmd/config.go

```diff
@@ -14,11 +14,19 @@ var once sync.Once
 
 func InitSDKConfig() {
 	once.Do(func() {
-		initSDKConfig()
+		checkOrInitSDKConfig()
 	})
 }
 
-func initSDKConfig() {
+func checkOrInitSDKConfig() {
```

Comment:
```suggestion
// checkOrInitSDKConfig updates the prefixes for all account types and seals the config.
// DEV_NOTE: Due to the separation of this repo and the SDK, where the config is also sealed,
// we have an added check to return early in case the config has already been set to the expected
// value. 
func checkOrInitSDKConfig() {
```